### PR TITLE
Delay redis test

### PR DESCRIPTION
**Pull request by ipapapa**

### CODE REVIEW

File: scripts/dynomite/dyn_redis_test.py [hunk 1]
```diff
@@ -229,7 +234,11 @@ def main():
                       dest="numkeys",
                       default="100",
                       help="Number of keys. Default is 100\n")
-   
+    parser.add_option("-d", "--delay",
+                      action="store",
+   		      dest="delay",
+                      default="0",
+                      help="Delay between operations. Default is 0\n")
```
Comment (shailesh33) [inline scripts/dynomite/dyn_redis_test.py]: is this is milliseconds?

Comment (ipapapa) [inline scripts/dynomite/dyn_redis_test.py]: time.sleep(secs)

Comment (shailesh33) [inline scripts/dynomite/dyn_redis_test.py]: I meant, can you add that in the description?

Comment (ipapapa) [inline scripts/dynomite/dyn_redis_test.py]: done

**Pull request merged**